src/modules/site-v2/templates/order/order.html

```diff
@@ -232,7 +232,7 @@ <h5 class="modal-title" id="deleteModalTitle">Are you sure you want to delete th
 
     $('#deleteItem').click(function(e){
         e.preventDefault()
-        const data = {csrf_token: $('#csrf_token').val(), itemToRemove: $(this).attr('info')}
+        const data = {csrf_token: $('#csrf_token').val(), itemToRemove: {name: $(this).data('value').name, species: $(this).data('value').species}}

```

Comment:
I had no idea you could access HTML `data` fields as objects like this. Very cool! :)